### ISSUE TITLE
FocusZone tabbing doesn't announce aria information

### ISSUE THREAD

**Issue by rickyp-ms** [OfficeDev/office-ui-fabric-react#2468]
### Bug Report

- __Package version(s)__: Master
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: (if applicable)

#### Describe the issue:

As I tab through the example FocusZone on the example app (e.g. npm start) it seems that I'm not getting any aria information correctly. I found this while making the change https://github.com/OfficeDev/office-ui-fabric-react/pull/2467 which adds aria information to the example focuszone.

#### Actual behavior:

Narrator announces "right arrow" but instead used to announce "group"

#### Expected behavior:

For narrator to announce the aria information for the element being arrowed to

### If applicable, please provide a codepen repro:

If you look at https://github.com/OfficeDev/office-ui-fabric-react/pull/2467 I updated the photo example so it should narrate out what index item you are currently focused on. Once that goes in, it should be pretty easy to repro on any repo.

**Comment by rickyp-ms** [comment]
This is how it should work
https://codepen.io/anon/pen/WEZaRj
